services/system/Transact/src/RTransact.cpp

```diff
@@ -930,6 +943,12 @@ std::optional<HttpReply> RTransact::serveSys(const psibase::HttpRequest& request
          return reply;
       }
    }
+   else if (target == "/jwt_key" && request.method == "GET")
+   {
+      check(user.has_value(), "Unauthorized");
+      check(to<XAdmin>().isAdmin(*user), "Forbidden");
```

Comment:
Since `XAdmin` service is a subjective service, can this inline action only be called in an RPC or otherwise subjective context?

Reply:
Right. The chain theoretically could make it work by providing an `x-admin` service. Otherwise, `x-admin` won't be found in an objective context.